src/blackdoor/cqbe/node/Node.java

```diff
@@ -39,9 +40,19 @@ private void startServer(int port) {
 		serverThread = new Thread(server);
 		serverThread.start();
 	}
-	
-	private void configureAddressing(L3Address wanAddress){
-		me = wanAddress;
+
+	private void configureAddressing(int port, boolean local) throws Exception {
```

Comment:
well then we need to somehow work on that, since it's not clear how getLocalHost decides which interface to return an IP for (for me it returns the ip of a vmware virtual ethernet adapter).